pandas/tests/series/test_timeseries.py

```diff
@@ -33,6 +33,19 @@ def _simple_ts(start, end, freq='D'):
     return Series(np.random.randn(len(rng)), index=rng)
 
 
+def test_NA_values_with_cache():
+    # GH 22305
+    na_values = [None, np.nan, pd.NaT]
```

Comment:
There should be a fixture for this you can use from conftest. Should also eliminate need for looping

Reply:
@WillAyd : This is not immediately trivial.  Note that @anmyachev is using the same "fixture" twice for the loop.

I don't know off the top of my head how you can use the same fixture twice (without declaring it with two separate variables).  Any ideas?

Reply:
IIRC you can alias the fixture

Reply:
In a similar situation I've used

https://github.com/realead/pandas/blob/356b8aab754c32e469508e23c447d8c650938843/pandas/conftest.py#L298

Here is an example: https://github.com/pandas-dev/pandas/blob/9caf58f3ee712a26b4ac9b9a2501141844a455ce/pandas/tests/indexes/test_base.py#L1454-L1467

Reply:
Test is using `unique_nulls_fixture` now. Thanks @realead for example.